README.md

```diff
@@ -171,12 +172,14 @@ or later, which officially supported C++11.
 3. HuC firmware is needed for AVC/HEVC/VP9 low power encoding bitrate control, including CBR, VBR, etc. As of now, HuC firmware support is disabled in Linux kernels by default. Please, refer to i915 kernel mode driver documentation to learn how to enable it. Mind that HuC firmware support presents in the following kernels for the specified platforms:
    * APL/KBL: starting from kernel 4.11
    * CFL: starting from kernel 4.15
+   * ICL: starting from kernel 4.8
```

Comment:
You are saying that ICL appeared _before_ CFL?! That's kind of impossible from my perpective. Refer to: https://github.com/Intel-Media-SDK/MediaSDK/wiki/Intel-Graphics-Support-in-Linux-Kernels#all-kernels. ICL support in general appeared only in 4.17 kernel under alpha support.